src/main/resources/templates/forgot-password.html

```diff
@@ -21,6 +21,8 @@
     <link rel="stylesheet"
           href="/webjars/startbootstrap-sb-admin-2/4.0.2/css/sb-admin-2.min.css" />
 
+    <link href="/css/stylesheet.css" rel="stylesheet" type="text/css" />
```

Comment:
In the future, maybe we can refactor this so that forgot-password, login & register all extend the same layout.